projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts

```diff
@@ -0,0 +1,13 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Textarea`, () => {
```

Comment:
```suggestion
test.describe(`Checkbox`, () => {
```

Reply:
Oh sorry for that 😅, addressed this comment